src/Explorer/Explorer.tsx

```diff
@@ -1088,6 +1209,24 @@ export default class Explorer {
       );
   }
 
+  public getDownloadModalConent(fileName: string): JSX.Element {
```

Comment:
>getDownloadModalConent [](http://example.com/codeflow?start=9&length=22)

Why is this in explorer.tsx?

I know sometime back - there was a conscious effort to  make Explorer.tsx lean.

please check this pr in and fix it immediately. dont block the checkin for this.